cmd/expired-authz-purger/main.go

```diff
@@ -0,0 +1,152 @@
+// Copyright 2014 ISRG.  All rights reserved
```

Comment:
Are you okay with just removing the header here? We previously had a conversation that  the main license file was fine on it's own and I've been meaning to go through and trim the rest of these out in a PR for a bit...